src/Adapter/PlatformServices.Interface/ObjectModel/ITestMethod.cs

```diff
@@ -46,7 +47,7 @@ public interface ITestMethod
         /// Gets the fully specified method name metadata format.
         /// </summary>
         /// <example>
-        ///     <code>MethodName`2(ParamTypeA,ParamTypeB,�)</code>
+        ///     <code>MethodName`2(ParamTypeA,ParamTypeB,�)</code>
```

Comment:
Broken encoding?

Reply:
Every time I save that file, VS Code breaks the encoding... Thanks for pointing it out.